testdata/validate-data/knora-api-subset.ttl

```diff
@@ -0,0 +1,27 @@
+@prefix knora-api: <http://api.knora.org/ontology/knora-api/v2#> .
```

Comment:
I need the inheritance information for the queries. This file is only used during tests. In the production the knora-api ontology returned from the API during runtime is used.